docs/src/pages/guides/probes.md

```diff
@@ -148,6 +148,151 @@ Currently, Monika only supports Content-Type value `application/x-www-form-urlen
 
 By default, the request body will be treated as-is. If the request header's `Content-Type` is set to `application/x-www-form-urlencoded`, it will be serialized into URL-safe string in UTF-8 encoding.
 
+## Fake Data
+
+You can use fake data to pass through your URLs, request body, and request headers. Here is the list of available fake data:
+
+- `{{ alpha <count> }}` returns a alphabet characters random string with the length of the `count`.
+
+  The default value for `count` is 8.
+
+  e.g `{{ alpha }}` returns `abcdefgh`, `{{ alpha 3 }}` returns `abcd`
+
+- `{{ alphaNumeric <count> }}` returns an alphanumeric characters random string with the length of `count`.
+
+  The default value for `count` is 8.
+
+  e.g `{{ alphaNumeric }}` returns `ab12ef34`, `{{ alphaNumeric 4 }}` returns `ab12`
+
+- `{{ countryCode }}` returns a country code
+
+  e.g `{{ countryCode }}` returns `US` or `ID`
+
+- `{{ color }}` returns a color
+
+  e.g `{{ color }}` returns `lime` or `green`
+
+- `{{ currency }}` returns a currency
+
+  e.g `{{ currency }}` returns `USD, IDR, JPY`
+
+- `{{ email }}` returns an email
+
+  e.g `{{ email }}` returns `monika@hyperjump.tech`
+
+- `{{ fullName }}` returns a full name
+
+  e.g `{{ fullName }}` returns `John Doe`
+
+- `{{ gender }}` returns a gender
+
+  e.g `{{ gender }}` returns `Male/Female`
+
+- `{{ isostring }}` returns a date in ISO String format
```

Comment:
this is not random date right? but the date of the request? if so, `returns the date of the request sending in ISO format`